data/json/snippets/mutant_anatomy.json

```
@@ -158,6 +160,54 @@
       "As you cut into its flesh you are disturbed by its sickly-sweet smell, like fermenting fruit"
     ]
   },
+  {
+    "type": "snippet",
+    "category": "<bug_harvest_centipede_chitin>",
+    "text": [
+      "Its legs contain no flesh, just sinews hanging in yellow-green ichor",
```

```suggestion
      "Its legs contain no muscles, just sinews hanging in yellow-green ichor",
```
"Flesh" is a bit general.